codes/ruby/chapter_sorting/heap_sort.rb

```diff
@@ -0,0 +1,43 @@
+=begin
+File: heap_sort.rb
+Created Time: 2024-04-10
+Author: junminhong (junminhong1110@gmail.com)
+=end
+
+### 堆的长度为 n ，从节点 i 开始，从顶至底堆化 ###
+def sift_down(nums, n, i)
+  while true
+      # 判断节点 i, l, r 中值最大的节点，记为 ma
```

Comment:
Please reduce the code to 2-space indentation

Reply:
thanks, 
[89273fa](https://github.com/krahets/hello-algo/pull/1261/commits/89273fa70b19350c5ecdb96f6b6c64754594142f)